src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -337,6 +337,7 @@ public SnowflakeStreamingIngestChannelInternal<?> openChannel(OpenChannelRequest
       payload.put("schema", request.getSchemaName());
       payload.put("write_mode", Constants.WriteMode.CLOUD_STORAGE.name());
       payload.put("role", this.role);
+      payload.put("is_iceberg", isIcebergMode);
```

Comment:
no request payload test coverage that needed to be updated?

Reply:
I don't think we have unit test for this. Tried delete some entries and still pass unit tests. As for IT, I think we should wait for the change on sfctest0.

Reply:
Please add backlog JIRAs that we need test coverage on whichever front you've made changes but didn't have to also update any tests to make them pass.

Reply:
Added [here](https://snowflakecomputing.atlassian.net/browse/SNOW-1490151)